grammars/toml.cson

```diff
@@ -3,37 +3,74 @@
 'fileTypes': ['toml']
 'patterns': [
   {
-    'match': '(?:^\\s*)(\\[\\[([^\\]]+)\\]\\])'
-    'captures':
-      '2': 'name': 'variable.keygroup.array.toml'
-    'name': 'keygroup.toml'
+    'begin': '(?:^\\s*)(\\[\\[)'
+    'beginCaptures':
+      '1': 'name': 'punctuation.definition.table.array.begin.toml'
+    'end': '\\]\\]'
+    'endCaptures':
+      '0': 'name': 'punctuation.definition.table.array.end.toml'
+    'name': 'entity.name.section.table.array.toml'
```

Comment:
This will include any leading whitespace in the `begin` capture as well.  That's undesirable, so a better option would be to add this name to capture 0 of `beginCaptures`, `contentName`, and capture 0 of `endCaptures` (while making the existing 0 a 1).